src/collectd.conf.in

```diff
@@ -2084,9 +2084,12 @@
 #<Plugin write_redis>
 #	<Node "example">
 #		Host "localhost"
-#		Port "6379"
-#		Timeout 1000
-#		Prefix "collectd/"
+#		Port 6379
+#		Database 0
+#		Timeout 0
+#		Retention 0 # infinite
+#		StoreRates true
+#		Prefix ""
```

Comment:
The timeout change was unintentional, reverted.

I have removed the default prefix because I felt it make it unclear how to use the plugin without any prefix.